### PR TITLE
FR14: Added feature of editing tracks

### PR DESCRIPTION
**Pull request by lkyostila**
Feature: Edit, remove and add tracks.

### CODE REVIEW

File: front/debug.log [hunk 1]
```diff
@@ -0,0 +1 @@
+[1020/131729.354:ERROR:directory_reader_win.cc(43)] FindFirstFile: M��ritetty� polkua ei l�ydy. (0x3)
```
Comment (Liljenmaa) [inline front/debug.log:1]: Add a new commit and remove this file

File: front/src/dayview/Dayview.css [hunk 2]
```diff
@@ -53,13 +53,56 @@
...
+}
+
 .dayviewContainer .sevenGrid {
   border:solid 1px black;
 }
 
-/* To have 7 boxes force size 100 / 7 = 14.28571428571429 */
+/* To have 7 boxes force size 100% / 7 = 14.28571428571429% */
 .dayviewContainer .sevenGrid > div {
     max-width:14.28571428571429%;
```
Comment (Liljenmaa) [inline front/src/dayview/Dayview.css:75]: Let's remove that magic number

max-width: calc(1 / 7 * 100%)

Comment (Liljenmaa) [inline front/src/dayview/Dayview.css:75]: Let's remove that magic number

max-width: calc(1 / 7 * 100%)

File: front/src/dayview/Dayview.css [hunk 3]
```diff
@@ -53,13 +53,56 @@
...
+    flex-direction: row;
+}
+
+.dayviewContainer .eightGrid {
+  border:solid 1px black;
+}
+
+/* To have 8 boxes force size 100% / 8 = 12.5% */
+.dayviewContainer .eightGrid > div {
+    max-width:12.5%;
```
Comment (Liljenmaa) [inline front/src/dayview/Dayview.css:88]: You could do the same here if you want to

File: front/src/dayview/Dayview.css [hunk 4]
```diff
@@ -53,13 +53,56 @@
...
+    flex-direction: row;
+}
+
+.dayviewContainer .tenGrid {
+  border:solid 1px black;
+}
+
+/* To have 10 boxes force size 100% / 5 = 20% */
+.dayviewContainer .tenGrid > div {
+    max-width:20%;
```
Comment (Liljenmaa) [inline front/src/dayview/Dayview.css:101]: And here

File: front/src/dayview/Dayview.js [hunk 5]
```diff
@@ -148,20 +148,35 @@ class Dayview extends Component {
             key={key}
             name={props.tracks[key].name}
             state={props.tracks[key].trackSupervision}
-	    notice={props.tracks[key].notice}
+	          notice={props.tracks[key].notice}
```
Comment (Liljenmaa) [inline front/src/dayview/Dayview.js:151]: Tab error, check indentation

File: front/src/dayview/Dayview.js [hunk 6]
```diff
@@ -148,20 +148,35 @@ class Dayview extends Component {
...
             //TODO final react routing
             to={"/trackview/"+props.date.toISOString()+"/" + props.tracks[key].name}
           />
         );
       }
-
+      if (items.length == 6) {
```
Comment (Liljenmaa) [inline front/src/dayview/Dayview.js:157]: Use switch-case

Comment (Liljenmaa) [reply front/src/dayview/Dayview.js:157]: Or better yet, use an array of these "xGrids" and loop over that array by doing something like var classNames = grids[items.length]

**Comment by TuoLuo994** [comment]
Make new branch from current develop and do the same changes with comment above.

**Pull request closed without merging**